dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -89,3 +131,56 @@ CWWKD1010.unknown.entity.prop.explanation=The repository method is attempting \
  to use an entity property that does not exist.
 CWWKD1010.unknown.entity.prop.useraction=Update the repository method to match \
  a valid property that is defined by the entity.
+
+CWWKD1011.unknown.method.pattern=CWWKD1011E: The {0} method of the {1} repository \
+ interface does not match any of the patterns defined by Jakarta Data. \
+ A repository method must either use annotations such as {2} to define operations, \
+ be a resource accessor method with no parameters and returning one of {3}, \
+ or the method must be named according to the Query by Method Name pattern. \
+ Method names for Query by Method Name must begin with one of the {4} keywords, \
+ followed by 0 or more additional characters, optionally followed by the \
+ 'By' keyword and one or more conditions delimited by 'And' or 'Or' keywords. \
+ Some examples of valid repository method names are: {5}.
+CWWKD1011.unknown.method.pattern.explanation=Repository methods must follow the \
+ patterns that are defined by Jakarta Data.
+CWWKD1011.unknown.method.pattern.useraction=Update the repository method to \
+ adhere to one of the patterns from Jakarta Data.
+
+CWWKD1012.fd.missing.param.anno=CWWKD1012E: Parameter {0} of the {1} method of \
+ the {2} repository interface does not match an entity property. Parameters that \
+ do not match an entity property must be annotated with the By annotation or have \
+ one of the special parameter types: {3}. If the parameter is intended to define \
+ a condition for the restriction of query results, annotate the parameter with \
+ the By annotation and assign the value of the By annotation to be the entity \
+ property name. Alternatively, enable the -parameters compile option and give \
+ the parameter the same name as the entity property.
+CWWKD1012.fd.missing.param.anno.explanation=Parameters of repository find and \
+ delete methods must correspond to entity properties or be special parameter \
+ types that are defined by Jakarta Data.
+CWWKD1012.fd.missing.param.anno.useraction=Update the repository method such \
+ that all parameters correspond to entity properties or have one of the Jakarta \
+ Data special parameter types.
+
+CWWKD1013.cde.missing.param.anno=CWWKD1013E: Parameter {0} of the {1} method of \
+ the {2} repository interface does not match an entity property. Parameters that \
+ do not match an entity property must be annotated with the By annotation and \
+ assign the value of the By annotation to be the entity property name. \
+ Alternatively, enable the -parameters compile option and give the parameter \
+ the same name as the entity property.
+CWWKD1013.cde.missing.param.anno.explanation=Parameters of repository count, \
+ delete, and exists methods must correspond to entity properties.
+CWWKD1013.cde.missing.param.anno.useraction=Update the repository method such \
+ that all parameters correspond to entity properties.
+
+CWWKD1014.upd.missing.param.anno=CWWKD1014E: Parameter {0} of the {1} method of \
```

Comment:
CWWKD1014.upd.missing.param.anno=CWWKD1014E: Parameter {0} of the {1} method of the {2} repository interface does not match an entity property. Parameters that do not match an entity property must have one of the parameter annotations: {3}. 

CWWKD1014.upd.missing.param.anno.explanation=Parameters of repository update methods must correspond to entity properties.

CWWKD1014.upd.missing.param.anno.useraction=Update the repository method such that all parameters correspond to entity properties. To define a condition for the restriction of query results, annotate the parameter with the By annotation and assign the value of the By annotation to be the entity property name. Alternatively, enable the -parameters compile option and give the parameter the same name as the entity property.